### ISSUE TITLE
Suggestion: Add support for speaker mute button?

### ISSUE THREAD

**Issue by TonyAki** [valinet/ThinkPadLEDControl#4]
Sorry for the double post, but another thing that's bothering me a bit is that there doesn't seem to be support for the speaker mute button on T480s. Any chance that'll get added? I tried every ID in debug from 00 to FF, and none seemed to do the trick. Maybe it has a different offset value (along with the caps lock key which also didn't respond to any IDs), but I don't know what I'm doing well enough to try that

**Comment by valinet** [comment]
There's a similar issue regarding support of newer models, it seems something has changed in the mean time, but unfortunately, I don't have the time to develop new functionality and the devices to test it on at the moment. May I suggest, you take a look at the source and try to patch in the required stuff so that you get it working? Information about stuff like this can usually be found in the ThinkPad modules from the Linux kernel, which usually has interfaces for using the features of these laptops.

Sorry, but unfortunately, this is the most I can say at the moment.

**Comment by TonyAki** [comment]
Sounds good, I'll look into it, appreciate the quick responses :)

**Comment by valinet** [comment]
Yeah, I am proud too, I get mail push notifications for Github on my iPhone from my very own mail server I set up and host myself. I am amazed when I actually see all the pieces working together.

**Comment by TonyAki** [comment]
Ahaha, all the work we put in to get the little things working just the way we want. I know the feeling

**Comment by valinet** [comment]
Indeed, and when you set stuff on your own, you can do things the usual commercial free stuff do not offer you. Like, on iOS, I like the stock Mail app, but Gmail dors not have push notifications for it, only on its own app. With this setup, I no longer have such a limitation, which is great.
